src/vs/workbench/contrib/preferences/browser/settingsWidgets.ts

```diff
@@ -665,6 +734,10 @@ export class ObjectSettingWidget extends AbstractListSettingWidget<IObjectDataIt
 		const rowElement = $('.setting-list-edit-row');
 
 		const keyWidget = this.renderEditWidget(item.key, rowElement);
+
+		// We have only rendered the key
+		rowElement.querySelector('.setting-list-object-input')?.classList.add('setting-list-object-input-key');
```

Comment:
This is a little suspect to me, can we just add the class when the element is rendered?

Reply:
We cannot add it when the element is rendered as the function that renders the key (`renderEditWidget`) is responsible for rendering both keys and values to keep things dry. The ways to get around it would be:

- pass in a flag in these functions, indicating if the thing being rendered is a key or a value, or
- make separate functions for rendering keys and values, which may be less dry

What do you think is better?